pkgs/top-level/all-packages.nix

```diff
@@ -12,25 +12,8 @@ with pkgs;
 
 {
 
-  # Allow callPackage to fill in the pkgs argument
   inherit pkgs;
 
-
-  # Override system. This is useful to build i686 packages on x86_64-linux.
-  forceSystem = system: kernel: nixpkgsFun {
```

Comment:
@matthewbauer I think all `nixpkgsFun` uses from `all-packages.nix` can be removed with this.

Reply:
Awesome - there was one left for xbursttools that we can replace with nixpkgsCross.